qiskit/synthesis/linear_phase/cz_depth_lnn.py

```diff
@@ -132,7 +150,7 @@ def _create_patterns(n):
     return pats
 
 
-def synth_cz_depth_line_mr(mat: np.ndarray) -> QuantumCircuit:
+def synth_cz_depth_line_mr(mat: np.ndarray, use_dag: bool = False) -> QuantumCircuit | DAGCircuit:
```

Comment:
Added `use_dag` to the function because it's used in  `synth_clifford_depth_lnn`.